### ISSUE TITLE
Wordle Game

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/Play-With-Python#23]
### Play With Python Repository (Proposing new issue)
:red_circle: **Project Title** : Wordle Game</br>
:red_circle: **Aim** : Create a Wordle game using Python language.</br>

**********************************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by sidling1** [comment]
Can I attempt this, my approach is -
Generate a Random 6 letter word and a set of alphabets containing each letter of the word and some other alphabets,
And fix the number of moves that can be made.

**Comment by abhisheks008** [comment]
@sidling1 one issue at a time. You can get this assigned after finishing your first one.

**Comment by sidling1** [comment]
I completed my first one , now can I do this one?

**Comment by abhisheks008** [comment]
Yeah you can. But in this particular game try to implement `tkinter` to create a GUI for the users so that it looks more appealing than the command prompt version.

Issue assigned to you @sidling1

**Comment by abhisheks008** [comment]
Updates please @sidling1
